lib/puppet/provider/package/brewcask.rb

```diff
@@ -45,15 +45,20 @@ def query
   end
 
   def install
+    install_cmd = ['brew']
     if install_options.any?
-      execute ["brew", "install", "Caskroom/cask/#{resource[:name]}", *install_options].flatten, command_opts
+      install_cmd << 'cask install'
     else
-      execute ["brew", "boxen-install", "Caskroom/cask/#{resource[:name]}"], command_opts
+      install_cmd << 'boxen-cask-install'
     end
+    install_cmd << install_options if install_options.any?
```

Comment:
This line should be able to go under the existing `if install_options.any?` a couple of lines up. Should become something like this and save the extra inline expression:

```rb
def install
  install_cmd = ['brew']

  if install_options.any?
    install_cmd << 'cask install'
    install_cmd << install_options
  else
    install_cmd << 'boxen-cask-install'
  end
  
  install_cmd << resource[:name]

  execute install_cmd.flatten, command_opts
end
```